base_tier_validation/views/tier_definition_view.xml

```diff
@@ -21,34 +24,59 @@
-                        <field name="model_id" options="{'no_create': True}"/>
-                        <field name="review_type"/>
-                        <field name="reviewer_id"
-                               attrs="{'invisible': [('review_type', '!=', 'individual')]}"/>
-                        <field name="reviewer_group_id"
-                               attrs="{'invisible': [('review_type', '!=', 'group')]}"/>
+                <sheet>
+                    <div class="oe_button_box" name="button_box">
+                        <button name="toggle_active" type="object" class="oe_stat_button" icon="fa-archive">
+                            <field name="active" widget="boolean_button"/>
+                        </button>
+                    </div>
+                    <group>
+                        <group name="left">
+                            <field name="model_id" options="{'no_create': True}"/>
+                            <field name="review_type"/>
+                            <field name="reviewer_id"
+                                   attrs="{'invisible': [('review_type', '!=', 'individual')]}"/>
+                            <field name="reviewer_group_id"
+                                   attrs="{'invisible': [('review_type', '!=', 'group')]}"/>
+                        </group>
+                        <group name="right">
+                            <field name="company_id" groups="base.group_multi_company" options="{'no_create': True}"/>
+                            <field name="sequence"/>
+                        </group>
                     </group>
-                    <group name="right">
-                        <field name="company_id" groups="base.group_multi_company" options="{'no_create': True}"/>
-                        <field name="sequence"/>
-                        <field name="active"/>
+                    <group col="4" name="bottom">
+                        <field name="python_code" colspan="4"/>
                     </group>
-                </group>
-                <group col="4" name="bottom">
-                    <field name="python_code" colspan="4"/>
-                </group>
+                </sheet>
             </form>
         </field>
     </record>
 
+    <record id="tier_definition_view_search" model="ir.ui.view">
+        <field name="name">tier.definition.search</field>
+        <field name="model">tier.definition</field>
+        <field name="arch" type="xml">
+            <search>
+                <field name="model_id"/>
+                <field name="reviewer_id"/>
+                <field name="reviewer_group_id"/>
+                <field name="active"/>
+                <separator/>
+                <filter string="All" name="all" domain="['|', ('active', '=', False), ('active', '=', True)]" />
+                <group expand="0" string="Group By">
+                    <filter string="Model" domain="[]" context="{'group_by':'model_id'}"/>
```

Comment:
I'd propose to add a `name` attribute to the group by `filter`, since without it it:
- It can't be selected with a xpath.
- It can't be used with `default_search_foo` on an action context.

[[source](https://github.com/odoo/odoo/commit/8f88c0336f881dd59d7c25b64c5bcba1989f998b#diff-150872ec366afb816e1fe98612a57634)]

And it's required starting with Odoo 12.